### PR TITLE
Added Info Page and Cougarettes Logo

### LINKED ISSUE #285: Host CougarCS Information Packet on CougarCS Website
[Information Packet](https://docs.google.com/document/d/19VczLYr3j1rrIl337YlBcWzCp8TBCKrHqhCJmQ6TGNs/edit)

This packet has been a critical part of our member onboard and communication strategy. It has proven immense value in nearly every interaction with students interested to learn about our club.

Rather than attach a PDF to every email and direct people to a Google Drive folder, let's host this on www.cougarcs.com/info (for example). When people go to this link, it loads the PDF file nice and easy.

This solves a few problems:
- We don't need to attach a PDF file to communications
- We can easily direct people to information
- The link is fixed. We can update the file freely and not worry about QR codes or whatever downstream

I achieve a similar effect on my personal website (nykolas.me) with my resume. It's just a PDF in the website public HTML folder and I directly link it.

### LINKED ISSUE #304: Add Cougarettes to About Page Partners
Hi team, could we add Cougarettes to our About Page at the bottom in "Partner Clubs" section? See an example below. 

image

Cougarettes Logo
https://drive.google.com/file/d/18NLOgNhTLr0kJOVHjCrJ05Sh0a2uROJI/view?usp=sharing

Redirect Link (if user clicks image)
https://uhcougarettes.com/

### PR DESCRIPTION
**Pull request by LeftAutomated**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [X] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR adds this feature.
Added 'Info' to the navigation bar
Added 'Info' page with viewable pdf file
Added clickable Cougarettes Logo in About page 

## Related Tickets & Documents

Fixes #285 
Fixes #304

## Mobile & Desktop Screenshots/Recordings

CougarCSInfo
CougarCSAboutPartnerClubs

## Added tests

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 docs
- [ ] 📕 storybook
- [X] 🙅 no documentation needed

### CODE REVIEW

**Review by shubhsheth — CHANGES_REQUESTED** [review]
@LeftAutomated  This doesn't necessarily need to be a page. We just need to have cougarcs.com/info to show the pdf or redirect to the pdf file. You don't have to link it in the navigation bar either.

The purpose of the #285 is to have a permanent hosting space for the Info packet and an easy to remember URL.

File: src/pages/Info/Info.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const meta = {
+	title: 'Info',
+	desc: 'CougarCS Information Packet',
+	url: 'https://cougarcs.com/info',
+	img: 'https://i.ibb.co/NTLFrdj/cougarcs-background11.jpg',
+};
+
+const Info = () => {
+	return (
+		<div>
```
Comment (vyas0189) [inline src/pages/Info/Info.js:14]: Can u make this div full view height of the page?

File: src/pages/Info/Info.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+			<MetaData {...meta} />
+			<iframe src={infoPDF} title='csPDF' width='100%' height='700px' />
```
Comment (vyas0189) [inline src/pages/Info/Info.js:16]: This iframe can be 100% of the parent div

**Review by vyas0189 — CHANGES_REQUESTED** [review]

File: src/pages/Info/Info.js (hunk 1 above)
Comment (crudemandarin) [reply src/pages/Info/Info.js:14]: Hmm how do you mean? I kinda like how @LeftAutomated includes the website nav at the top of the page. It lets people we directed to the PDF hop right into our website.

image

However, I can absolutely see the point that maybe we shouldn't include the nav and just display the PDF full screen. I could go either way, what do you guys think? @vyas0189 @KevFlo @shubhsheth @LeftAutomated

Comment (vyas0189) [reply src/pages/Info/Info.js:14]: image

Comment (vyas0189) [reply src/pages/Info/Info.js:14]: His current shows the footer in the page view

**Comment by vyas0189** [comment]
You can keep the footer @LeftAutomated

File: src/pages/Info/Info.js (hunk 1 above)
Comment (KevFlo) [reply src/pages/Info/Info.js]: To be honest, I do like how this looks without the footer, it looks more streamlined. While with the footer it looks more professional/ structured if that makes any sense.  Maybe if we can make the footer on this page alone like 25% smaller i'd be 
 game.

**Comment by KevFlo** [comment]
Testing this on a phone leads to the pdf being downloaded and not displayed. The point of this page is for people to scan the QR code and having the pdf display on their phones. @vyas0189 @crudemandarin  do you guys have any suggestions?

**Comment by KevFlo** [comment]
Screen Shot 2021-10-04 at 8 59 55 AM

**Comment by LeftAutomated** [comment]
For mobile, it might be better for the user to download the pdf file instead of embedding the pdf.
Desktop Version:
image
Mobile Version:
image

**Comment by KevFlo** [comment]
> For mobile, it might be better for the user to download the pdf file instead of embedding the pdf. Desktop Version: image Mobile Version: image

@LeftAutomated I see you are putting the pdf on the assets folder, why not try using just the pdf google drive link? 
phone pdf
hmmm


Here are two things I found on the subject, ill test this out more over the weekend if you aren't available to do so.
Let me know if you need anything.

**Pull request merged**